activiti-spring-boot-starter/src/test/java/org/activiti/spring/boot/tasks/TaskRuntimeDeleteTaskTest.java

```diff
@@ -22,27 +24,37 @@
 
 @RunWith(SpringRunner.class)
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.NONE)
-@ContextConfiguration
-@FixMethodOrder(MethodSorters.NAME_ASCENDING)
 public class TaskRuntimeDeleteTaskTest {
 
-    private static String currentTaskId;
     @Autowired
     private TaskRuntime taskRuntime;
     @Autowired
     private TaskAdminRuntime taskAdminRuntime;
     @Autowired
-    private SecurityManager securityManager;
+    private SecurityUtil securityUtil;
 
 
+    @After
+    public void cleanUpWithAdmin() {
+        securityUtil.logInAs("admin");
```

Comment:
This clean up block could be moved to a bean that's autowired in the different tests.